app/services/irs_attempts_api/tracker_events.rb

```diff
@@ -158,8 +168,8 @@ def mfa_login_phone_otp_submitted(reauthentication:, success:)
     end
 
     # Tracks when the user has attempted to log in with the piv cac MFA method to their account
-    # @param [String] subject_dn
     # @param [Boolean] success
+    # @param [String] subject_dn
```

Comment:
IMHO I wouldn't fix this unless you need to edit something else in this PR anyway, but I think in the merge conflict this line got moved from above to below `success`.

Reply:
So, the merge conflict removed the top two lines entirely 😨. I put them back in a commit, but noticed that the params were out of order, so I moved subject_dn below success on purpose, to match the function definition. Was that the right thing to do?

Reply:
Oh, my bad! Yes, good catch. 👍